src/components/layout/skeletonGrid.js

```diff
@@ -0,0 +1,59 @@
+import { Box, Skeleton, SkeletonCircle, SkeletonText } from '@chakra-ui/react'
+import React from 'react'
+
+export const SkeletonGrid = ({ type }) => {
+  if (type === 'CategoryFilter') {
+    return (
+      <Box padding='6' boxShadow='lg' bg='white'>
```

Comment:
```suggestion
      <Box padding='6' boxShadow='lg'>
```

Please make sure if the actual grid has a background color.